res/css/views/settings/encryption/_DeleteKeyStoragePanel.pcss

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+.mx_DeleteKeyStoragePanel {
+    .mx_DeleteKeyStoragePanel_content {
+        display: flex;
+        flex-direction: column;
+        gap: var(--cpd-space-3x);
+
+        > span {
+            font: var(--cpd-font-body-md-medium);
+            text-align: center;
+        }
+    }
+
+    .mx_DeleteKeyStoragePanel_footer {
+        display: flex;
+        flex-direction: column;
+        gap: var(--cpd-space-4x);
+        justify-content: center;
+    }
+}
```

Comment:
This is a complete duplication of `_ResetIdentityPanerl.pcss`, I wonder if in this case we can use one style sheet like `_DestructiveComponent.pcss` and uses it in the three components. But this solution is going against our code style guide.

Reply:
Yeah, it felt very wrong - I wasn't sure if that was more or less wrong than breaking the pattern with one common css file for two components, but I guess it is probably better than duplication.